### ISSUE TITLE
We should alert when rendering errors occur in chat

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#252191]
For screen reader users, we need to alert when an error comes in as the response. 

![Image](https://github.com/user-attachments/assets/e61e6015-2411-4e95-b4be-158440997d19)

**Comment by meganrogge** [comment]
Also, errors are not keyboard focusable, but should be
